# Message for controller output
#

geometry_msgs/Vector3 AngVelRef
geometry_msgs/Vector3 AngComp
geometry_msgs/Vector3 AngRef
geometry_msgs/Vector3 Torq
geometry_msgs/Vector3 KaiErr
geometry_msgs/Vector3 Sigma
float32 Kbar
float32 Time
